libraries/AP_RCProtocol/AP_RCProtocol_CRSF.h

```diff
@@ -116,6 +119,13 @@ class AP_RCProtocol_CRSF : public AP_RCProtocol_Backend {
         CRSF_COMMAND_RX_BIND = 0x01,
     };
 
+    // SubType IDs for CRSF_FRAMETYPE_CUSTOM_TELEM
+    enum CustomTelemSubTypeID : uint8_t {
+        CRSF_CUSTOM_TELEM_PASSTHROUGH = 0xF0,
+        CRSF_CUSTOM_TELEM_STATUS_TEXT = 0xF1,
+        CRSF_CUSTOM_TELEM_PASSTHROUGH_ARRAY = 0xF2,
+    };
```

Comment:
preexisting, but these are really ugly... should be:
```suggestion
    enum class CustomTelemSubType : uint8_t {
        PASSTHROUGH = 0xF0,
        STATUS_TEXT = 0xF1,
        PASSTHROUGH_ARRAY = 0xF2,
    };
```

And users go from `CRSF_CUSTOM_TELEM_PASSTHROUGH` to `CustomTelemSubType::PASSTHROUGH`.

Not for this PR, 'though.

Reply:
yeah, probably best to do it in a separate PR where we refactor all similar enums

Reply:
@peterbarker I really don't agree with the use of enum class where you are representing numbers that actually go on the wire - it's far to inconvenient and adds little value IMO other than lots of casting.